test-data/unit/stubgen.test

```diff
@@ -2688,3 +2688,20 @@ def f():
     return 0
 [out]
 def f(): ...
+
+[case testIncludeDocstrings]
```

Comment:
We also need the same test without `--include-docstrings` option to verify that docstrings are not copied.